docs/contributing/getting-started-developers.md

```diff
@@ -98,7 +101,11 @@ More info:
 Install [Debugger for Chrome extension](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome) in Visual Studio Code debug Wizard Client.
 After starting the client using `yarn start` in VSCode Debug View (Ctrl+Shift+D) select "Debug WebTS Client" and start debugging (F5)
 
-More info:
+#### Troubleshooting:
```

Comment:
This will be fixed in version 0.5, you should be able to install nightly and local version side by side

Reply:
true !!